atomic_reactor/plugins/pre_resolve_remote_source.py

```diff
@@ -63,6 +65,22 @@ def json_filename(cls, name: Optional[str]):
             return REMOTE_SOURCE_JSON_FILENAME
 
 
+def transform_results(results):
+    if not results or not isinstance(results, list):
+        return
```

Comment:
This check should be unnecessary, if a plugin returns `None`, all the metadata decorators exit without doing anything